service/s3/eventstream_test.go

```diff
@@ -146,24 +60,22 @@ func TestSelectObjectContentEventStream(t *testing.T) {
 	}
 }
 
-func TestSelectObjectContentEventStream_Close(t *testing.T) {
-	_, eventMsgs := mockSelectObjectContentEventStreamsOutputEvents()
+func TestSelectObjectContent_ReadClose(t *testing.T) {
+	_, eventMsgs := mockSelectObjectContentReadEvents()
 	sess, cleanupFn, err := eventstreamtest.SetupEventStreamSession(t,
 		eventstreamtest.ServeEventStream{
 			T:      t,
 			Events: eventMsgs,
 		},
-		false,
+		true,
```

Comment:
See if the operation is HTTP/2 and if it is able to be determined.

Reply:
Doesn't look like it.